downloads/downloads-impl/src/main/java/com/duckduckgo/downloads/impl/DefaultFileDownloadNotificationManager.kt

```diff
@@ -77,7 +77,7 @@ class DefaultFileDownloadNotificationManager @Inject constructor(
             .setContentText("$filename ($progress%).")
             .setShowWhen(false)
             .setSmallIcon(R.drawable.ic_file_download_white_24dp)
-            .setProgress(100, progress, progress == SUMMARY_ID)
+            .setProgress(100, progress, progress == 0)
```

Comment:
This should be `0` but when I used the IDE to create the `SUMMARY_ID` constant, as it also had the value `0`, the IDE also replaced this instance. Reverting that now